tasks/linter.py

```diff
@@ -393,3 +395,28 @@ def gitlab_ci(_, test="all", custom_context=None):
         if not res["valid"]:
             print(color_message(f"Errors: {res['errors']}", "red"), file=sys.stderr)
             raise Exit(code=1)
+
+
+@task
+def releasenote(ctx):
+    """
+    Lint release notes with Reno
+    """
+    branch = os.environ.get("BRANCH_NAME")
+    pr_id = os.environ.get("PR_ID")
+
+    run_check = is_pr_context(branch, pr_id, "release note")
+    if run_check:
+        github = GithubAPI(repository=GITHUB_REPO_NAME, public_repo=True)
+        if github.is_release_note_needed(pr_id):
+            if not github.contains_release_note(pr_id):
+                print(
+                    "Error: No releasenote was found for this PR. Please add one using 'reno'"
+                    ", see https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno"
+                    ", or apply the label 'changelog/no-changelog' to the PR."
+                )
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏We can display it on stderr / in red